src/test.ts

```diff
@@ -19,7 +19,4 @@ getTestBed().initTestEnvironment(
   BrowserDynamicTestingModule,
   platformBrowserDynamicTesting()
 );
-// Then we find all the tests.
-const context = require.context('./', true, /\.spec\.ts$/);
-// And load the modules.
-context.keys().map(context);
+
```

Comment:
making sure this was intentional

Reply:
This was intentional. There was a change made to Karma that makes these lines unnecessary